src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,63 @@
+import sys
```

Comment:
How is this script `get_product_lts_version.py` ultimately called?  I expected to see it in `helm_install.sh` but I dont? 🤔

Reply:
Im also not sure how this script `get_product_lts_version.py` is ultimately used?  I expected to see it in `helm_install.sh` but I dont? 🤔

Reply:
This scripts fetch all releases of product, filter Enterprise editions (LTS versions) and get the latest version by sorting them and picking the first one. 

Also this script is called by HelmTestPlanSpec.java (through helm_install_chart) in bamboo plan (see https://stash.atlassian.com/projects/CSI/repos/kubernetes-ci/pull-requests/151/overview)

Reply:
Got ya, so `Enterprise` == `LTS`